### PR TITLE
[TA] Add known issues

### PR DESCRIPTION
**Pull request by maririos**
- `CategoriesFilter` not working on analyze https://github.com/Azure/azure-sdk-for-net/issues/19237.
- `Statistics` for `AnalyzeBatchActionsResult` are not currently returned even if the user passes `IncludeStatistics  = true`. https://github.com/Azure/azure-sdk-for-net/issues/19268

### CODE REVIEW

**Review by iscai-msft — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeBatchActionsResult.cs [hunk 1]
```diff
@@ -56,7 +56,10 @@ internal AnalyzeBatchActionsResult(AnalyzeJobState jobState, IDictionary<string,
         public IReadOnlyCollection<RecognizeLinkedEntitiesActionResult> RecognizeLinkedEntitiesActionsResults { get; }
 
         /// <summary>
-        /// <summary> if showStats=true was specified in the request this field will contain information about the document payload. </summary>
+        /// Gets statistics about the operation executed and how it was processed
+        /// by the service. This property currently won't return a value even if
+        /// <see cref="AnalyzeBatchActionsOptions.IncludeStatistics"/> to true.
```
Comment (suhas92) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeBatchActionsResult.cs:61]:
```suggestion
        /// <see cref="AnalyzeBatchActionsOptions.IncludeStatistics"/> is set to true.
```

**Review by suhas92 — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,10 @@
 ### Fixes
 - `RecognizePiiEntities` and `TextAnalyticsActions.RecognizePiiEntitiesOptions` were always passing `PiiEntityDomainType.PHI`. Now, it is only passed when requested by the user [19086](https://github.com/Azure/azure-sdk-for-net/issues/19086).
 
+### Known Issues
+- The parameter `CategoriesFilter` in `RecognizePiiEntitiesOptions` doesn't have an impact when using it in `StartAnalyzeBatchActions`. [19237](https://github.com/Azure/azure-sdk-for-net/issues/19237).
```
Comment (deyaaeldeen) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:28]: NIT:
```suggestion
- The parameter `CategoriesFilter` in `RecognizePiiEntitiesOptions` is currently not working when used in `StartAnalyzeBatchActions`. [19237](https://github.com/Azure/azure-sdk-for-net/issues/19237).
```

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActions.cs [hunk 3]
```diff
@@ -5,31 +5,32 @@
 
 namespace Azure.AI.TextAnalytics
 {
-    /// <summary> Determines the list of actions to be passed as arguments for AnalyzeBatchActionsOperation class. </summary>
+    /// <summary> Determines the set of actions to execute on the input documents.</summary>
```
Comment (deyaaeldeen) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActions.cs:8]: NIT:
```suggestion
    /// <summary> Determines the set of actions that will get executed on the input documents.</summary>
```

**Review by deyaaeldeen — APPROVED** [review]
Consider adding a bullet point for the healthcare method not working with AAD for now. For example: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/textanalytics/ai-text-analytics#known-issues.

**Pull request merged**